source/2019-12-17-countdown-to-the-new-year-2019-ember-leaflet.md

```diff
@@ -0,0 +1,27 @@
+---
+title: Countdown to The New Year: ember-leaflet
+author: Jessica Jordan
+tags: 2019, Addons, DecEmber
+alias: 2019/12/17/countdown-to-the-new-year-2019-ember-leaflet
+responsive: true
+---
+
+The first of its kind "Countdown to The New Year: 31 Days of Ember Addons" series highlights popular addons in the Ember community. We hope you enjoy learning about fun tools for Ember developers together with us each day!
+
+## Day 17
+
+In the 17th increment of **Countdown to The New Year** let's take a look at an exciting Ember addon to add **maps** to our apps: **ember-leaflet**!
+
+Learn more about [ember-leaflet on Ember Observer](https://emberobserver.com/addons/ember-leaflet).
+
```

Comment:
add `READMORE`

(am not sure if this is helpful to repeat, lmk)

Reply:
Thank you, that's definitely useful - I can update the other posts accordingly as well!